sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -25,6 +26,19 @@ public Discount getDiscount(@PathVariable Long id) {
         return service.getAll();
     }
 
+    //Возвращаю при пустом поле поиска или отсутсвие заданых критериев пока пустой список
```

Comment:
Please return all items if `searchWork` is empty. Also please don't forget to remove this comment

Reply:
Done